addons/web/static/tests/views/list_view_tests.js

```diff
@@ -7959,6 +7959,63 @@ QUnit.module("Views", (hooks) => {
         }
     );
 
+    QUnit.test(
+        "empty editable list with sample data: create and duplicate record",
```

Comment:
this test looks very specific. I would have expected a more "basic" test for the feature (I'm not saying you should remove this one, I guess you had your reasons to write it that way): have a list with n records, select all, duplicate.